sykmeldingalerts-gcp-dev.yaml

```diff
@@ -11,14 +11,14 @@ spec:
       channel: '#sykmelding-alerts-dev'
   alerts:
     - alert: syfosm-asynkron-app-nede
-      expr: kube_deployment_status_replicas_available{deployment=~"syfosmaltinn"} == 0
+      expr: kube_deployment_status_replicas_available{deployment=~"syfosmaltinn|sykmeldinger-backend-proxy"} == 0
```

Comment:
kanskje lage en egen for apper som ikke er async?

Reply:
enten det, eller bare fjerne "asynkron"? har det noen hensikt å skille på disse?

Reply:
Hensikten med å skille var vel at alarmen skulle trigge raskere hvis det var en app som noen har synkrone avhengigheter til :)

Reply:
den står på 5min nå for asynkrone apper.
tenker du noe ala 1-2min?